pkg/api/api.go

```diff
@@ -161,6 +161,12 @@ func ConfigureAPI(treeID uint) {
 	}
 	if viper.GetBool("enable_retrieve_api") || viper.GetBool("enable_stable_checkpoint") ||
 		slices.Contains(viper.GetStringSlice("enabled_api_endpoints"), "searchIndex") {
+		redisClient = redis.NewClient(&redis.Options{
```

Comment:
This shouldn't be needed, the provider will create the Redis client - https://github.com/sigstore/rekor/blob/main/pkg/indexstorage/indexstorage.go#L34

Reply:
ah, thanks for catching that.